src/Controller/PasswordController.php

```diff
@@ -31,8 +31,11 @@ public function changePassword(Request $request, UserRepository $userRepository)
 
         $newPassword = $request->get('new-password');
 
+        if (strlen($newPassword) < 6) {
+            $this->addFlash('danger', 'La taille du mot de passe doit etre supérieur à 6 caractères');
+        }
 
-        if (!empty($newPassword)) {
+        if (!empty($newPassword) && strlen($newPassword) >= 7) {
```

Comment:
The suspens is unbearable ! What happens when you have a password with EXACTLY 6 characters !!!!?????